Modified code

shashwat28 opened the pull request:
- added new processed directory for face cutouts
- extracting face cutouts to processed directory

ArvindSinghRawat commented:
Merge it with feature/face_detection, as this is not complete?
We need to add logic to export cutout of the images from bounding box

shashwat28 committed:
Modified code

- added new processed directory for face cutouts
- extracting face cutouts to processed directory

shashwat28 committed:
Correction

- Arguments now follow Snake Case rule

ArvindSinghRawat reviewed (changes requested):
Add a boolean parameter in `**kwargs`, for displaying the image. like ( display_result )
If it exists and its value is true, then only show the image, otherwise directly export the image.

shashwat28 committed:
Removed Processed Images

shashwat28 committed:
Refactored Code

- Modularised existing code
- Added `data/processed` in gitignore
- Removed redundant Bounding boxes

shashwat28 commented:
Done Requested changes,

Pending : **kwargs implementation
